### PR TITLE
add reach example

### PR DESCRIPTION
**Pull request by mforets**

### CODE REVIEW

**Comment by mforets** [comment]
in the generated html i still see the output even when using a colon `R = Algorithm1(A, δ, X0, μ, T);` ... strange

**Comment by schillic** [comment]
Yes, this seems to be the [behavior of `@example`](https://juliadocs.github.io/Documenter.jl/stable/man/syntax/#@example-block-1). I also realized this the last time and just removed the semicolons :)

**Comment by schillic** [comment]
The HTML page takes very long to load in my browser 🐌 😃

**Comment by mforets** [comment]
yes, thanks for checking. i've hidden the output by letting the last command to be the `plot`. 

from my side, this branch is ready to be merged.

**Comment by mforets** [comment]
thanks for the `#hide` tip.

File: docs/src/man/reach_zonotopes.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+
+    # bloating factors
+    Anorm = norm(A, Inf)
+    α = (expm(δ*Anorm) - 1 - δ*Anorm)/norm(X0, Inf)
+    β = (expm(δ*Anorm) - 1)*μ/Anorm
+
+    # discretized system
+    n = size(A, 1)
+    ϕ = expm(δ*A)
+    N = floor(Int64, T/δ)
```
Comment (schillic) [inline docs/src/man/reach_zonotopes.md]: I prefer `Int`.

File: docs/src/man/reach_zonotopes.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+
+    # preallocate working vector and output
+    Q = Vector{LazySets.LazySet}(N)
+    R = Vector{LazySets.LazySet}(N)
+
+    # initial reach set in the time interval [0, δ]
+    ϕp = (I+ϕ)/2
+    ϕm = (I-ϕ)/2
+    Ballβ = BallInf(zeros(n), β)
```
Comment (schillic) [inline docs/src/man/reach_zonotopes.md]: Move this line (`Ballβ`) down to the next block?

File: docs/src/man/reach_zonotopes.md [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+
+```@contents
+Pages = ["reach_zonotopes.md"]
+Depth = 3
+```
+
+## Introduction
+
+In this section we present an algorithm implemented using `LazySets` that computes
+the reachable sets of a linear affine ordinary differential equation (ODE). This
```
[the remark this answers is not in the document]
Comment (mforets) [reply docs/src/man/reach_zonotopes.md]: fixed.

File: docs/src/man/reach_zonotopes.md [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+algorithm is from A. Girard's *"Reachability of uncertain linear systems using zonotopes*,
+HSCC. Vol. 5. 2005. We have chosen this algorithm for the purpose of illustration
+of a complete application of `LazySets`.
+
+## Algorithm
+
+```@example example_reach_zonotopes
+using LazySets, Plots
+
+function Algorithm1(A, δ, X0, μ, T)
```
[the remark this answers is not in the document]
Comment (mforets) [reply docs/src/man/reach_zonotopes.md]: done

**Comment by schillic** [comment]
Great example by the way, I like it 👍

**Comment by mforets** [comment]
thanks for your prompt feedback!

**Pull request merged**